main.py

```diff
@@ -121,31 +121,33 @@ def registration():
         with open("access_token.txt") as f:
             if access_token != f.read():
                 print("bad token")
-                flash('Incorrect access token', 'error')
+                flash("Incorrect access token", "error")
                 return render_template("registration.html")
 
         if "@" not in email:
-            flash('Incorrect email', 'error')
+            flash("Incorrect email", "error")
             return render_template("registration.html")
 
         if "." not in email:
```

Comment:
I think it might be easier to read for new contributors to understand if the email validation logic were split into another function. As far as I'm concerned, all it would need to do is return a `bool`.